contrib/epee/src/readline_buffer.cpp

```diff
@@ -16,6 +17,8 @@ static std::string last_prompt;
 std::mutex line_mutex, sync_mutex, process_mutex;
 std::condition_variable have_line;
 
+std::vector<std::string> rdln::readline_buffer::completion_commands = {"exit"};
```

Comment:
> FWIW I am not particularly worried about it since only readline is ever likely to call these functions

The functions that use these variables are all private to readline_buffer.cpp so I am not worried at all.